### PR TITLE
Added important links button to the organization information page

### PR DESCRIPTION
**Pull request by pnishant23**
In the organization information page we have the buttons which directs to various link of a particular organization. However not all the important links are getting fetched from the Open API. Hence those missing important links have been added to the _imp_links.json_ file created by @nishantwrp and I have fetched that file in the _compile-data.js_ file which feeds the data to organization information page. 

To added these important links head towards the _imp_links.json_ file in _/api_ folder and added the json object. 

Guide to add the links 
- The icon field in the object takes all the icons that are available in the [schemantic UI](https://react.semantic-ui.com/elements/icon/). Refer to the link to add the particular icon
- The title field appears as a tooltip when we hover on the icon button 

@nishantwrp, @beingnoble03  please review the code and let me know. If everything seems to be alright do merge the code so that this feature can be implemented on the live site

### CODE REVIEW

**Comment by pnishant23** [comment]
@nishantwrp please review this PR and let me know if this is mergeable or not OR should I create a merge request with master branch.

File: src/components/org-info.jsx [hunk 1]
```diff
@@ -177,6 +178,25 @@ const OrgInfo = ({ data }) => {
             }
           />
         )}
+        {data.links &&
+          data?.links?.map(item => {
```
Comment (nishantwrp) [inline src/components/org-info.jsx:182]: don't need optional chaining

File: package-lock.json
Comment (nishantwrp) [inline package-lock.json:1]: you can revert this change

File: api/compile-data.js [hunk 2]
```diff
@@ -241,7 +245,18 @@ const compileData = () => {
   const distinctOrganizations = organizationSet.extract()
   const gsocOrganizations = []
   distinctOrganizations.forEach(orgList => {
-    gsocOrganizations.push(getCombinedOrgJson(orgList))
+    // ---------new code ------------
```
Comment (nishantwrp) [inline api/compile-data.js:248]: can you please clean these comments.

**Review by nishantwrp — COMMENTED** [review]
Thanks @pnishant23 for taking this up! Took an initial pass, will take a more thorough look once the deploy preview is ready.

**Comment by nishantwrp** [comment]
Hi @pnishant23, the number of lines

**Comment by nishantwrp** [comment]
Just to give some context, I was trying to get netlify deploy preview to work. Seems like changes to package-lock.json & yarn-lock.json need to be reverted.

File: src/components/org-info.jsx (hunk 1 above)
Comment (pnishant23) [reply src/components/org-info.jsx:182]: Ok, will remove that

File: api/compile-data.js (hunk 2 above)
Comment (pnishant23) [reply api/compile-data.js]: i have removed this comments form the file

**Comment by pnishant23** [comment]
closing this PR because created an updated PR for this which is [PR 148](https://github.com/nishantwrp/gsoc-organizations/pull/148)
